### DISCUSSION TITLE
Cuphead

### DISCUSSION THREAD

**Discussion by burtvadr** [skyline-emu/skyline#517] [Game Requests]
cuphead is in hype and is a light game that the community is looking forward to porting to android

**Comment by PixelyIon** [comment]
God help whoever is going to play Cuphead with OSC but it is a good game so I might look into it at some point.

**Reply by burtvadr** [reply]
idk, but i I've seen like over 15 cuphead fan ports for mobile, not kidding

**Reply by burtvadr** [reply]
unfinished ports**

**Reply by KristalFragmentz** [reply]
I'm sure it would be more worth it's salt to experience the Switch version. Especially with the DLC rounding the corner.

**Comment by PixelyIon** [comment]
Well, it works now as of the latest builds.

**Reply by ArvinJai** [reply]
But does the saves get corrupted if the emulator crashes? I don't see my saves any time I restart the game?? Sometimes I would Exit to title (from the map screen) and then check for my save but I get a black screen re-entering the Main menu... Restart the entire game and no Saves are stored. I'm using a Samsung Galaxy S22 Ultra (12 GB RAM and the Adreno GPU 730) with the latest build.

**Reply by SuDanix82** [reply]
Hi! Have you found  a solution with save  games?
Have the same problem with Tab S8 Ultra

**Reply by Draoolia** [reply]
Need to wait more updates.... Skyline not work well :(

**Reply by ghost** [reply]
Is it still laggy like before? I mean game used to lag as I progressed in-game.
